csharp/private/BUILD

```diff
@@ -1,13 +1,10 @@
 load(":toolchains.bzl", "configure_toolchain")
 
-exports_files(["nunit/shim.cs"])
+exports_files(["nunit/shim.cs", "wrappers/dotnet.cc"])
```

Comment:
Why is this being exported?

Reply:
The file `dotnet.cc` is used for wrapping the `dotnet` executable, you can see the template reference [here](https://github.com/Brightspace/rules_csharp/blob/cb981c78d77bd79887c19e4fdab9bbece967778f/csharp/private/rules/wrapper.bzl#L1).